Source/WebKit/WPEPlatform/wpe/WPEMonitor.cpp

```diff
@@ -480,7 +480,6 @@ gdouble wpe_monitor_get_scale(WPEMonitor* monitor)
 void wpe_monitor_set_scale(WPEMonitor* monitor, double scale)
 {
     g_return_if_fail(WPE_IS_MONITOR(monitor));
-    g_return_if_fail(scale >= 1);
```

Comment:
I guess we could keep the check, but change it to ensure scale is greater than 0 (like we already do in WPEToplevel)